test/functions/generic/poi/fieldDifferentModDynamicDisp.chpl

```diff
@@ -0,0 +1,55 @@
+// Tests calling a method on a field when the field's type pretends to be the
+// parent type.
+module One {
+  class OuterClass {
+    var field;
+
+    proc one(x) {
+      field.two(x);
+    }
+  }
+}
+
+module Two {
+  private use TwoPoint1;
+
+  class Stored : StoredParent {
+
+    override proc two(x) {
+      writeln("In Stored.two with value ", x);
+    }
+  }
+}
+
+module TwoPoint1 {
+  class StoredParent {
+    proc two(x) {
+      writeln("In StoredParent.two with value ", x);
+    }
```

Comment:
To test `private use TwoPoint1` in module Two, I propose adding `proc twoPoint1` here, then invoking it from proc one(). It should be visible to class Stored because class StoredParent is visible.

Reply:
Isn't that already handled by the inheritance test?  What distinguishes it from the "three" method there?

Reply:
You are right, that covers it. Never mind.